### ISSUE TITLE
Error in man page ganeti-rapi

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#172]
Originally reported of Google Code with ID 116.

```
What software version are you running? 
2.1.6

<b>What steps will reproduce the problem?</b>
man ganeti-rapi

<b>What is the expected output? What do you see instead?</b>
man page mentions the location of the rapi_users file like this

'' The users and their rights are defined in a file named rapi_users, located in the /var/ganeti directory. ''

The correct directory is $STATEDIR/lib/ganeti which by default would point the file to:

 /var/lib/ganeti/rapi_users


I got stuck half a day on auth problems because of that :-)



<b>Please provide any additional information below.</b>

```
Originally added on 2010-08-16 13:10:00 +0000 UTC.

Attachments: 
[20100816150942.png](https://storage.googleapis.com/google-code-attachments/ganeti/issue-116/comment-0/20100816150942.png)

**Comment by Ganeti-Issues-Migrator** [comment]
```
BTW, please note that ganeti doesn't log anything which points to this problem, even when running the rapi daemon with debug option.
It would be nice to log about the unexisting rapi_user file.
```
Originally added on 2010-08-16 13:12:13 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in commit 215e40a9519c89, thanks for the report!
```
Originally added on 2010-08-16 14:10:17 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:08 +0000 UTC.

Changed State: Released
